config/config_test.go

```diff
@@ -41,12 +41,15 @@ agents:
 	assert.Equal(t, "my-artifactory-key", cfg.Artifactory.Key)
 	assert.Equal(t, "my-agent-name", cfg.Agents[0].Name)
 	assert.Equal(t, "s3", cfg.Agents[0].Downloader.Type)
-	assert.Equal(t, "my-s3-bucket", cfg.Agents[0].Downloader.AwsBucket)
-	assert.Equal(t, "my-prefix", cfg.Agents[0].Downloader.AwsPrefix)
-	assert.Equal(t, "my-aws-key", cfg.Agents[0].Downloader.AwsKey)
-	assert.Equal(t, "my-aws-secret", cfg.Agents[0].Downloader.AwsSecret)
-	assert.Equal(t, "us-west-2", cfg.Agents[0].Downloader.AwsRegion)
 	assert.Equal(t, 900, cfg.Agents[0].SleepDuration)
 	assert.Equal(t, "my-repo", cfg.Agents[0].ArtifactoryRepo)
 
+	expectedConfig := map[interface{}]interface{}{
```

Comment:
Shouldn't this be map[string]interface{} ?

Reply:
Good catch, this should be `map[interface{}]interface{}`, but there are other places in the tests where I used `map[string]interface{}`.  Those should be changed to `map[interface{}]interface{}`